### ISSUE TITLE
Port Narrator announcement for StringCollectionEditor items

### ISSUE THREAD

**Issue by M-Lipin** [dotnet/winforms#2188]
* .NET Core Version: 3.1
* Have you experienced this same bug with .NET Framework?: Yes

**Problem description:**
There is no screen reader announcement for item navigation and selecting in StringCollectionEditor UI.

**Actual behavior:**
Narrator does not announce the StringCollectionEditor item when navigating up and down thru the items list.

**Expected behavior:**
Narrator announces the item when focus goes to item when navigating using keyboard/mouse.

**Minimal repro:**
* Create WinForms app with ComboBox or another list control which collection can be edited using StringCollectionEditor.
* Add PropertyGrid control to the form and associate its selected object with previously added ComboBox.
* Run the app try to edit the collection of items (add few items and navigate up/down using keyboard within StringCollectionEditor textbox.
* Observe no announcement when navigation.

**Comment by merriemcgaw** [comment]
@Olina-Zhang can you confirm this is working and then close the issue if this is fixed.

**Comment by Olina-Zhang** [comment]
Verified this issue in the latest .Net 6.0 SDK build, this issue cannot repro now. Narrator can announce the items when using up/down arrow keyboard within StringCollectionEditor textbox.
![image](https://user-images.githubusercontent.com/26474449/102579533-985a6600-4137-11eb-8183-6483c51234f9.png)
